nvblox_ros/src/lib/mapper_initialization.cpp

```diff
@@ -184,60 +210,89 @@ void initializeMapper(const std::string& mapper_name, Mapper* mapper_ptr,
         occupied_region_half_width_m);
     mapper_ptr->lidar_occupancy_integrator().occupied_region_half_width_m(
         occupied_region_half_width_m);
-  }
+  }else{
+    ROS_ERROR_STREAM("occupied_region_half_width_m not found");
+    isInitializationSuccessful = false;
+  } 
 
   float free_region_decay_probability = 0.0f;
   if (nh.getParam("free_region_decay_probability",
                   free_region_decay_probability)) {
     mapper_ptr->occupancy_decay_integrator().free_region_decay_probability(
         free_region_decay_probability);
-  }
+  }else{
+    ROS_ERROR_STREAM("free_region_decay_probability not found");
+    isInitializationSuccessful = false;
+  } 
 
   float occupied_region_decay_probability = 0.0f;
   if (nh.getParam("occupied_region_decay_probability",
                   occupied_region_decay_probability)) {
     mapper_ptr->occupancy_decay_integrator().occupied_region_decay_probability(
         occupied_region_decay_probability);
-  }
+  }else{
+    ROS_ERROR_STREAM("occupied_region_decay_probability not found");
+    isInitializationSuccessful = false;
+  } 
 
   float mesh_integrator_min_weight = 0.0f;
   if (nh.getParam("mesh_integrator_min_weight", mesh_integrator_min_weight)) {
     mapper_ptr->mesh_integrator().min_weight(mesh_integrator_min_weight);
-  }
+  }else{
+    ROS_ERROR_STREAM("mesh_integrator_min_weight not found");
+    isInitializationSuccessful = false;
+  } 
 
-  bool mesh_integrator_weld_vertices = 0.0f;
+  bool mesh_integrator_weld_vertices = false;
   if (nh.getParam("mesh_integrator_weld_vertices",
                   mesh_integrator_weld_vertices)) {
     mapper_ptr->mesh_integrator().weld_vertices(mesh_integrator_weld_vertices);
-  }
+  }else{
+    ROS_ERROR_STREAM("mesh_integrator_weld_vertices not found");
```

Comment:
Shouldn't fail just because we don't overwrite the defaults. Trust the backend defaults! Keep them! I'll change this.